### ISSUE TITLE
Crash on BrowserActivity.java line 7

### ISSUE THREAD

**Issue by superdiazzz** [teads/TeadsSDK-android#147]
Hi Teads Team, 
I have upgrade version of teads dependency in this version:

```
implementation('tv.teads.sdk.android:sdk:4.3.0@aar'){
 transitive = true
}
```

![image](https://user-images.githubusercontent.com/10770867/68557771-6ae9b600-0469-11ea-84f2-f32986bdb298.png)

but crash in BrowserActivity still occured. Here what console log said:

> Fatal Exception: android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.intent.action.VIEW dat=intent://id/web/pages/wow/i/id/marketing/Mkt_lp_v4?trigger_item=591802550&venture=id&entrance=mktlp&laz_trackid=2:mm_150040307_51200739_2010250736:clk5qkv8f1dp7u907v26s0&dsource=sml&exlaz=e_Li3fREjGj2jChAM6aUDLfFaE%2BUKYYPbDMVkZMICDnX1nws9Lp%2FUuM4DrX0MXtJD1LkIvZSagg4EUe4kdT1DwXdAjA6e0jVeImzkNp1SjzCrPZyuBjUbw6aKeViiMmTBxwl70ynTQLQUAGvdtBhJtFayJYVMgRWODp%2FJtRTzSPEHWeLm0h7Oauh1hyyMvAOS8NcQZtb09QgTrq%2B2630FoXn9s7hCE9QozfEpuL1gCKt27oRyo2geFDf0wEikD%2F7lqP41YtOLNzIx%2BawLrZJcxt0FWAW42edS09BCOLDlncvX5g%2BJNurFIKe%2BLMByUPpzsccQTv9MxjmKuPtTvdAF7H1smSRbae%2BDCbSvarBq6D4iv%2F%2Fl30SqaHdf8l16NWp6S7qmJxwCgactm7k0vzsT9x7ZEGE8N41d689mgYd4pnKdMPaKY2gC9w%2FqrvvrFEveP%2FoxPvPHbWOG%2BHyPGbofvJP%2FPzhP0QtNKTweZN3Qgmw6MVcuc5gtZfB0HwxTSuvTH6ejkaPglHAAH2Bm5XIpS4G9derd%2BhdwZ1VYSJ51RGfSmwPZDInWYJcXsvrx0yXvMfm4368RaNmlRAMs2OaYRtejLt9P%2BmEkzNxbsAh%2B%2BvEvlfLBW5dHsYIVHCgJ5RKUh0NOAZmufjC6uUptLB8v%2F%2FrZfB0cFBfxhCJmAcEfBByYq3NHoxaUlzYkiwxLJEa9DGo3OAlRROXImxRUrm2RY93ntXXPGOJjZv%2FZZ7Gfx9ZmPnV2y8gNeddRL2cfMeOshcOViBJiYRjJ7FQQ%2BTKVXjdtiXqSuDzRATYTXcQTEhHUHGZG7AbOXbutPGvGXzNfflhPtg8rSNT5kJx1j3%2BOa7r%2BmvzHUtFdNF65rXXTZAWU%2Brpr4tDbhSBm10o5R8JNf1jDDLHPeY5hsHo%2BerIxWBd%2F8XeO8mJ1NmEzyBthA7N1pk6n8DqlTCWQ6vdYVD2KuKMaZa2uRAIbzXVsf50apsvh%2Ffwy60jGj%2BicKuTeXQb3Akehu3oIilZMkUzEFMUx4JzQsxMWeuv6MkZRXm4X01fU4jUcP0vX6KTAaZVIm6Zs%3D flg=0x10000000 }
>        at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:2014)
>        at android.app.Instrumentation.execStartActivity(Instrumentation.java:1675)
>        at android.app.Activity.startActivityForResult(Activity.java:4586)
>        at androidx.fragment.app.FragmentActivity.startActivityForResult(FragmentActivity.java:676)
>        at android.app.Activity.startActivityForResult(Activity.java:4544)
>        at androidx.fragment.app.FragmentActivity.startActivityForResult(FragmentActivity.java:663)
>        at android.app.Activity.startActivity(Activity.java:4905)
>        at android.app.Activity.startActivity(Activity.java:4873)
>        at tv.teads.sdk.android.engine.ui.browser.BrowserActivity$1.shouldOverrideUrlLoading(BrowserActivity.java:7)
>        at IG0.a(PG:1)
>        at org.chromium.android_webview.AwContentsClientBridge.shouldOverrideUrlLoading(PG:15)
>        at android.os.MessageQueue.nativePollOnce(MessageQueue.java)
>        at android.os.MessageQueue.next(MessageQueue.java:326)
>        at android.os.Looper.loop(Looper.java:160)
>        at android.app.ActivityThread.main(ActivityThread.java:6762)
>        at java.lang.reflect.Method.invoke(Method.java)
>        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
>        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)

Kindly help please

**Comment by guillaume-poi** [comment]
Hi everyone, 
We are facing a similar issue on our application.  It is happening on a different SDK version : 

`implementation("tv.teads.sdk.android:sdk:4.3.3@aar") {
        transitive = true
    }`

Here is the crash log : 

> Fatal Exception: android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.intent.action.VIEW dat=blob:https://ameli-cmd-front.damdy.com/4cd43685-5353-45df-949c-5c2b841da889 flg=0x10000000 }
       at android.app.Instrumentation.checkStartActivityResult + 1981(Instrumentation.java:1981)
       at android.app.Instrumentation.execStartActivity + 1636(Instrumentation.java:1636)
       at android.app.Activity.startActivityForResult + 4762(Activity.java:4762)
       at android.support.v4.app.FragmentActivity.startActivityForResult + 767(FragmentActivity.java:767)
       at android.app.Activity.startActivityForResult + 4702(Activity.java:4702)
       at android.support.v4.app.FragmentActivity.startActivityForResult + 754(FragmentActivity.java:754)
       at android.app.Activity.startActivity + 5123(Activity.java:5123)
       at android.app.Activity.startActivity + 5091(Activity.java:5091)
       at tv.teads.sdk.android.engine.ui.browser.BrowserActivity$1.shouldOverrideUrlLoading + 7(BrowserActivity.java:7)
       at org.chromium.android_webview.AwContentsClientBridge.shouldOverrideUrlLoading + 16(PG:16)
       at android.os.MessageQueue.nativePollOnce(MessageQueue.java)
       at android.os.MessageQueue.next + 379(MessageQueue.java:379)
       at android.os.Looper.loop + 144(Looper.java:144)
       at android.app.ActivityThread.main + 7529(ActivityThread.java:7529)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.Zygote$MethodAndArgsCaller.run + 245(Zygote.java:245)
       at com.android.internal.os.ZygoteInit.main + 921(ZygoteInit.java:921)

It appears from android version 7 to 9 on a wide range of devices. 
It's our second biggest crash source, any insight on this one ? 
Your help would be appreciated, thank you.

**Comment by benjaminvolland** [comment]
Hi @superdiazzz ,@guillaume-poignant,
Thanks for the report. I will take a look at this issue and I will come back soon.

**Comment by benjaminvolland** [comment]
Hi @superdiazzz and @guillaume-poignant,
The fix is available in the version 4.6.0 https://github.com/teads/TeadsSDK-android/releases/tag/v4.6.0
